lighthouse-core/audits/non-composited-animations.js

```diff
@@ -0,0 +1,148 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a diagnostic LH audit that provides details on animations that are not composited. */
+  title: 'Avoid non-composited animations',
+  /** Description of a diagnostic LH audit that shows the user animations that are not composited. */
+  description: 'Animations which are not composited can be janky and contribute to CLS. ' +
+    '[Learn more](https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count)',
+  /** [ICU Syntax] Label identifying the number of animations that are not composited. */
+  displayValue: `{itemCount, plural,
+  =1 {# animated element found}
+  other {# animated elements found}
+  }`,
+};
+
+const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
+
+/**
+ * Each failure reason is represented by a bit flag. The bit shift operator '<<' is used to define which bit corresponds to each failure reason.
+ * https://source.chromium.org/search?q=f:compositor_animations.h%20%22enum%20FailureReason%22
+ * @type {{flag: number, text: string}[]}
+ */
+const ACTIONABLE_FAILURE_REASONS = [
+  {
+    flag: 1 << 13,
+    text: 'Unsupported CSS Property',
```

Comment:
should we make this i18n'd? seems doable before we mingled the ID. I'm not sure how to work it in with the concatenation of the that. @paulirish do you feel the combination is worth losing other languages instead of another column?

not to reopen the can of worms that is our table discussions :)

Reply:
good call we should def i18n this string.  

when we start supporting more than 1 reason (in a separate PR), we can deal with this.. if we concat multiple reasons we can finally start using [`Intl.ListFormat()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat/ListFormat).  :D

Reply:
are you proposing keeping the name mingled in here with something like `Unsupported CSS Property ("{animationName}")`? 

I'm struggling a bit to see the value in interpolation there compared to a separate column, especially if other future reasons will be tied to the same animation. it also means anyone who wanted programmatic access to the name has to do locale-specific parsing to get it back out

Reply:
I also noticed that it might be a bit confusing what the animation name is vs the CSS property names:
```
Non-compositable CSS property: width ("fade-in")
```
Adding another column would reduce this confusion.

Reply:
I added the extra column for now.

When we do add the CSS Property names, how can I concatenate them with an i18n'd string?

Reply:
in that PR it'll look like 
```js
const UIStrings = {
  nonCompositablePropertyLabel: `Non-compositable CSS property: {propertyName}`,
}

str_(UIStrings.nonCompositablePropertyLabel, {propertyName})

```